### PR TITLE
feat(SPV-1043): Added clear filters button, added toaster

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: src/components/DateRangeFilter/DateRangeFilter.tsx [hunk 1]
```diff
@@ -122,6 +138,9 @@ export const DateRangeFilter = ({ withRevokedRange, className }: DateRangeFilter
         <Button onClick={onApplyDateRange} className="mt-4 w-full" variant="default">
           Apply
         </Button>
+        <Button onClick={onClearDateRange} className="mt-4 w-full" variant="secondary">
+          Clear
+        </Button>
```
Comment (chris-4chain) [inline src/components/DateRangeFilter/DateRangeFilter.tsx:143]: In the app (with the "filers" box closed) there should be some visual information that the filters are applied. 
> There is no info that some filters are applied.

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: src/components/DateRangeFilter/DateRangeFilter.tsx (hunk 1 above)
Comment (Nazarii-4chain) [reply src/components/DateRangeFilter/DateRangeFilter.tsx:143]: Added info message when filters are applied. What visual information I should add to display that filters are applied when the box is closed?

Comment (chris-4chain) [reply src/components/DateRangeFilter/DateRangeFilter.tsx:143]: This is example of the ideas:

- Use Clear Labels or Tags:
    - Display active filters as labels or tags above the content list.
    - Allow users to remove filters directly by clicking an 'X' on each tag.
    - Highlight Filter Icons or Buttons:

- Change the color or style of filter icons when they are active.
    - Add a small badge or indicator next to the icon showing the number of active filters.
    - Provide a Summary Bar:

- Include a concise summary that lists all active filters in one line.
    - Ensure it's easily noticeable but doesn't take up too much space.
    - Implement a 'Reset Filters' Option:

- Offer a single button to clear all filters, simplifying the reset process.
    - Place it where users expect to find it, such as near the filter options or at the top of the content area.
    - Avoid Overloading the Interface:

Comment (Nazarii-4chain) [reply src/components/DateRangeFilter/DateRangeFilter.tsx]: Added Icon

**Review by chris-4chain — APPROVED** [review]

File: src/components/DateRangeFilter/DateRangeFilter.tsx [hunk 2]
```diff
@@ -58,6 +90,11 @@ export const DateRangeFilter = ({ withRevokedRange, className }: DateRangeFilter
         <Button variant="outline">
           <ListFilter className="w-5 h-5" />
           <span className="ml-2">Filter</span>
+          {createdRange || updatedRange || revokedRange ? (
+            <span className="ml-2 bg-black/80 rounded-full text-white h-6 w-6 text-xs flex justify-center items-center">
+              +1
+            </span>
+          ) : null}
```
Comment (chris-4chain) [inline src/components/DateRangeFilter/DateRangeFilter.tsx:97]: After testing:
The `+1` content is not visually centered in the circle:
image

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: src/components/DateRangeFilter/DateRangeFilter.tsx (hunk 2 above)
Comment (Nazarii-4chain) [reply src/components/DateRangeFilter/DateRangeFilter.tsx]: Updated

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**